### PR TITLE
Updated readme to include setup for those using Apple silicon laptops

### PR DESCRIPTION
**Pull request by JakeGoodwin22**
## What does this pull request do?

Included set up guide for those running the latest apple chips as the current setup doesn't work.

## What is the intent behind these changes?

Make it easier for new starters to get set up by following a guide that is designed for their machine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,27 @@ bin/start-dependencies
...
+```
+docker-compose pull
+
+(nvm use)
+npm install
+
+Pull down following repositories:
+hmpps-assess-risks-and-needs, community-api, hmpps-auth, and offender-assessments-api
+
+Add '127.0.0.1 hmpps-auth' to your /etc/hosts file
```
Comment (sldblog) [inline README.md:45]: This is already mentioned above, I wonder if it's worth adding 'M1' sections in-between the 'general' sections? (Duplication is harder to maintain)

**Review by sldblog — COMMENTED** [review]
Thanks for doing this!
Comment about duplication

File: README.md (hunk 1 above)
Comment (JakeGoodwin22) [reply README.md]: Good point! I've changed the layout to make it clearer and to remove duplication

File: README.md [hunk 2]
```diff
@@ -27,10 +37,13 @@ You'll also need to add the following line to your `/etc/hosts` file:
 ### Running the app
 
 ```
-bin/start-dependencies
+If not using Apple chipset use: 'bin/start-dependencies'
```
Comment (sldblog) [inline README.md:40]: _not a blocker_
Is this because of the dockerize image to check for "is it up yet"? Otherwise it's using compose as well

**Review by sldblog — APPROVED** [review]
Looks good to me, no blockers.

@kth13, IIRC you have an M1 laptop too, could you review as well, please?

**Pull request merged**